perf_test/Benchmark_Context.hpp

```diff
@@ -95,6 +95,21 @@ inline void add_benchmark_context(bool verbose = false) {
   add_version_info();
 }
 
+struct Params {
+  static std::map<std::string, int> params_;
+
+  static void parse_inputs(int argc, char** argv) {
+    for (int i = 1; i < argc; i += 2) {
```

Comment:
This assumes something about all the arguments being of the form `--option value` and none being of the form `--use_option`